examples/logging/chatcompletion_sync.py

```diff
@@ -0,0 +1,22 @@
+import openai
+
+from log10.load import log10, log10_session
+
+
+log10(openai, DEBUG_=True, USE_ASYNC_=False)
+with log10_session():
```

Comment:
Will this keep working with the new setup?

Reply:
yes, tested and it works.